model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/RealmCacheSession.java

```diff
@@ -387,9 +387,6 @@ public RealmModel createRealm(String id, String name) {
     @Override
     public RealmModel getRealm(String id) {
         CachedRealm cached = cache.get(id, CachedRealm.class);
-        if (cached != null) {
```

Comment:
Are the changes in this file necessary? Could be omitted? This is the only collision with #7231

Reply:
Yes, it should not be a problem to omit this. I will revert it from this PR when working on https://issues.redhat.com/browse/KEYCLOAK-14806

Reply:
Thank you!